src/App.vue

```diff
@@ -806,20 +770,23 @@ export default {
       let createPlioResponse = await PlioAPIService.createPlio();
       this.$Progress.finish();
       if (createPlioResponse.status == 201) {
-        // once the plio is created, update it's settings as well
+        // once the plio is created, update its settings as well
         let plioUuid = createPlioResponse.data.uuid;
+        let settingsToUpdate = this.isPersonalWorkspace
+          ? this.userSettings.get("player")
+          : this.activeWorkspaceSettings.get("player");
         let updatePlioSettingsResponse = await PlioAPIService.updatePlioSettings(
```

Comment:
same point about using `as_map` or `as_object` to avoid redundant object -> map and map -> object conversion